### PR TITLE
COT-427 IAC Hearings Onboarding: Configure ExUI to point to IAC hearings endpoints

### PR DESCRIPTION
**Pull request by mwaheedanjum**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by rajeshthuraiyur — APPROVED** [review]
LGTM

File: config/default.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
       "caseAssignmentApi": "http://aac-manage-case-assignment-prod.service.core-compute-prod.internal"
     },
     "hearings": {
-      "hearingsJurisdictions": "SSCS,PRIVATELAW,CIVIL",
+      "hearingsJurisdictions": "SSCS,PRIVATELAW,CIVIL,IAC",
```
Comment (andywilkinshmcts) [inline config/default.json]: This should be IA, not IAC

File: config/default.json [hunk 2]
```diff
@@ -46,6 +46,10 @@
         "serviceApi": "http://civil-service-prod.service.core-compute-prod.internal",
         "caseTypes": "CIVIL"
       },
+      "iac": {
```
Comment (andywilkinshmcts) [inline config/default.json]: This should be "ia" not "iac"

**Review by andywilkinshmcts — DISMISSED** [review]

**Review by andywilkinshmcts — CHANGES_REQUESTED** [review]
Why does the pr include api/.yarn/install-state.gz
This shouldn't be checked in

**Comment by mwaheedanjum** [comment]
> Why does the pr include api/.yarn/install-state.gz This shouldn't be checked in

deleted

**Review by andywilkinshmcts — APPROVED** [review]

**Review by rajeshthuraiyur — APPROVED** [review]
LGTM

**Pull request merged**